modules/fts/pages/fts-advanced-settings-enableVerboseLogging.adoc

```diff
@@ -0,0 +1,9 @@
+= enableVerboseLogging
+
+The `enableVerboseLogging` setting enables additional logs for debugging purpose. Enabling this setting periodically adds more statistics whenever stats collection is done.
+
+The couchbase products support the following log tagging:source-highlighter: coderay
+
+* *INFO*: (default) - all logs tagged info, warn, fatal, crit
+* *WARN*: All logs tagged warn, fatal, crit
+* *DEBU*: All logs tagged debug, info, warn, fatal, crit
```

Comment:
No no no. Do not put log level information here.

enableVerboseLogging is only to collection additional statistics by the periodic stats collection task and the when the user hits this endpoint - /api/stats.